api/agent/server.go

```diff
@@ -610,16 +610,16 @@ func (s *Server) GetRouter() http.Handler {
 	apiRtr.HandleFunc("/api/tornjak/clusters/delete", s.clusterDelete)
 
```

Comment:
Can we actually move the `v1` APIs down here so we have all the v1 apis in one place?